.github/workflows/check.yml

```diff
@@ -13,19 +13,19 @@ jobs:
     steps:
       - uses: actions/checkout@v1
 
-      - name: Cache Cargo Registry
-        uses: actions/cache@v1
-        with:
-          path: ~/.cargo/registry
-          key: ${{ runner.os }}-wasm-registry-${{ hashFiles('src/rust/Cargo.lock') }}
-          restore-keys: ${{ runner.os }}-wasm-registry
-
-      - name: Cache Wasm
-        uses: actions/cache@v1
-        with:
-          path: src/rust/target
-          key: ${{ runner.os }}-wasm-build-${{ hashFiles('src/rust/Cargo.lock') }}
-          restore-keys: ${{ runner.os }}-wasm-build
+#      - name: Cache Cargo Registry
+#        uses: actions/cache@v1
+#        with:
+#          path: ~/.cargo/registry
+#          key: ${{ runner.os }}-wasm-registry-${{ hashFiles('src/rust/Cargo.lock') }}
+#          restore-keys: ${{ runner.os }}-wasm-registry
+#
+#      - name: Cache Wasm
+#        uses: actions/cache@v1
+#        with:
+#          path: src/rust/target
+#          key: ${{ runner.os }}-wasm-build-${{ hashFiles('src/rust/Cargo.lock') }}
+#          restore-keys: ${{ runner.os }}-wasm-build
```

Comment:
Please uncomment. The CI changes should not be a part of this PR.

Reply:
Shall I then create a separate PR ?

Reply:
And how often it happens? Is removing cache the only way of fixing this?

You can create a separate PR, and we continue the discussion there.

Reply:
It happened right now when i've reenabled caching :(

Reply:
https://github.com/enso-org/ide/pull/708